codegen/README.md

```diff
@@ -21,3 +24,9 @@ By running `./gradlew :sdk-codegen:build`, this package will:
    where `{filename}` is the relative filename of a model without ".json".
    For example, `codegen/sdk-codegen/build/smithyprojections/sdk-codegen/RDS-Data.2018-08-01/typescript-codegen`
    would contain the code generated for `aws-models/RDS-Data.2018-08-01.json`.
+
+## Troubleshooting
+
+1. Run `./gradlew :sdk-codegen:clean` to run a clean build
```

Comment:
```suggestion
1. Run `./gradlew :sdk-codegen:clean` to run clean task
```

Reply:
Running `./gradlew :sdk-codegen:clean` only runs the `clean` task.  This is helpful for cleaning up file from previous builds.